### ISSUE TITLE
S3 docs

### ISSUE THREAD

**Issue by kleenkanteen** [medusajs/medusa#4586]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

I just finished following the S3 plugin guide to store images: https://docs.medusajs.com/plugins/file-service/s3.

I have 2 suggestions:

1. In this part, I don't know what's the difference between the first half and bottom half. I used the top half which worked. When should one use the bottom version?:

![image](https://github.com/medusajs/medusa/assets/67878058/6feaf9c5-d943-4dcc-a0ec-4742f6afd4a1)

2. In this part:

![image](https://github.com/medusajs/medusa/assets/67878058/aec4686c-a7b6-4373-8ed4-b13dab146d05)

The proper url is <BUCKET_NAME>.s3.<S3_REGION>.amazonaws.com. I was stuck for 2 hours trying to figure out why my images are being saved to my S3 bucket after uploading them on the admin dashboard, but not showing up on the storefront. Turns out the region is also required or else the image gets blocked by the next server, it brings up a page with error 400: bad request.

**Comment by shahednasser** [comment]
Thank you @kleenkanteen for opening this issue! I'll look into it
